libs/editor/example/src/main/java/org/wordpress/android/editor/example/EditorExampleActivity.java

```diff
@@ -57,27 +61,58 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getIntent().getIntExtra(EDITOR_PARAM, USE_NEW_EDITOR) == USE_NEW_EDITOR) {
-            ToastUtils.showToast(this, R.string.starting_new_editor);
-            setContentView(R.layout.activity_new_editor);
+            ToastUtils.showToast(this, R.string.starting_aztec_editor);
+            setContentView(R.layout.activity_aztec_editor);
         } else {
-            ToastUtils.showToast(this, R.string.starting_legacy_editor);
-            setContentView(R.layout.activity_legacy_editor);
+            Fragment fragment = GutenbergEditorFragment.newInstance("", "", true, "us-en");
+            ToastUtils.showToast(this, R.string.starting_gutenberg_editor);
+            setContentView(R.layout.activity_gutenberg_editor);
+            FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
+            ft.replace(R.id.postEditor, fragment);
+            ft.commit();
         }
 
         mFailedUploads = new HashMap<>();
     }
 
     @Override
-    public void onAttachFragment(Fragment fragment) {
+    public void onAttachFragment(androidx.fragment.app.Fragment fragment) {
```

Comment:
I think that there is no need for a full package path as we already have import on the top of the file?

Reply:
Fixed in 0a7e0b6b6d